src/components/Forms/Input/Input.js

```diff
@@ -26,6 +28,8 @@ const Input = (props) => {
             placeholder={placeholder}
             onChange={handleChange}
             className={classes}
+            data-index={index}
```

Comment:
Where you have `data-index` can we make that `dataIndex` lets avoid `-` in the prop names, since they could be an issue later on

Reply:
Good point! I think I meant to include that temporarily. Will refactor this!

Reply:
I actually just realized that ``data-index`` is not the actual prop name - it's called ``index``. ``data-index`` is referred once when I set the value of my new prop name to ``index``!